.github/workflows/lint.yaml

```diff
@@ -1,14 +0,0 @@
-name: lint
-
-on: [push]
-
-jobs:
-  lint:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - uses: cachix/install-nix-action@v18
-        with:
-          nix_path: nixpkgs=channel:nixos-unstable
-      - run: nix-shell --run hn-rust-fmt-check
```

Comment:
why is this deleted?

if you put all the tasks in the same workflow then you have to wait longer because they can't run in parallel

Reply:
The workflow uses a flake now, no more `niv` and nixpkgs, which `hn-rust-fmt-check` was part of. `hn-rust-fmt-check` used to `fmt` and `clippy`, so the flake borrows these two commands with parameters from the holochain repo. It's not deleted, just replaced.

I thought that building the nix packages twice on two separate workflows would take longer, that's why I joined them. I split them again.

Reply:
And they run faster =)